v1/proto/service/gribi.proto

```diff
@@ -52,8 +52,12 @@ service gRIBI {
 message ModifyRequest {
   // A group of requests to add/modify/remove a single AFT entry.
   // gRIBI server relies on clients being conservative in what they send.
-  // As such, gRIBI server is expected to process the AFTOperations per the
-  // received order.
+  // As such, when processing AFTOperations, a gRIBI server is expected to:
+  //  * process AFTOperations per the received order.
+  //  * try not to close the RPC session due to errors in AFTOperation
```

Comment:
"try not to" vs. "not" -- do we want to be stronger here?

I think that in general the statement here should probably be:

```
errors that are encountered in a single AFTOperation should not
result in the Modify RPC session being closed. Rather, invalid
AFTOperations should be responded to with a failure within the
stream.
```

Reply:
ack. make sense.